www/package.json

```diff
@@ -4,23 +4,26 @@
   "scripts": {
     "start": "npm run generate-providers && docusaurus start",
     "build": "npm run generate-providers && docusaurus build",
+    "docusaurus": "docusaurus",
```

Comment:
I don't think we need this script, unless there's an explicit benefit from it? 💭

At the moment, this just hints whether there's an update available for the package 📦